Assets/Scripts/Saves/Serializers.cs

```diff
@@ -137,5 +131,106 @@ public class Weights : List<int>
         {
             public int[] Deserialize() => ToArray();
         }
+        
+        public class TeeSection : IVertex<DataTypes.TeeSection>
+        {
+            public string throughOrRight { get; set; }
+            public string throughOrLeft { get; set; }
+            public string leftOrRight { get; set; }
```

Comment:
Notwendigkeit die Getter und Setter explizit zu definieren?

Reply:
Die Library scanned standardmäßig nur nach Properties. (Außerdem hatten wir das auch schon so in #62 so gemacht, den hattest du doch reviewed?)